client/rest-high-level/src/test/java/org/elasticsearch/client/IngestClientIT.java

```diff
@@ -80,4 +86,76 @@ public void testDeletePipeline() throws IOException {
             execute(request, highLevelClient().ingest()::deletePipeline, highLevelClient().ingest()::deletePipelineAsync);
         assertTrue(response.isAcknowledged());
     }
+
+    public void testSimulatePipeline() throws IOException {
+        XContentType xContentType = randomFrom(XContentType.values());
+        XContentBuilder builder = XContentBuilder.builder(xContentType.xContent());
+        boolean isVerbose = randomBoolean();
```

Comment:
I think I have left a comment on this before, can we avoid randomizing values in integration tests and rather have two tests, one for verbose and one without verbose? Same for failures

Reply:
That means four tests?

Reply:
yep

Reply:
Okay. I am sorry I missed this comment from before. I separated it now.